messages/lightning.preview.site.md

```diff
@@ -1,38 +1,37 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+Enable Lightning Preview to see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when Lightning Preview detects source code changes.
```

Comment:
Note: to use this command on an existing experience site requires that your site has been published at least once after Lightning Preview has been enabled for your org.

Reply:
^ Can we say this in the dev guide page instead? It sounds like a consideration/limitation that we could add to the list in the LWC guide.